swan-lake/development-tutorials/test-document-the-code/test-ballerina-code/mocking.md

```diff
@@ -86,13 +81,14 @@ public function testGetRandomJoke() {
 }
 ```
 
-### Stub member functions and variables of an object
+### Stub the methods and fields of an object
```

Comment:
Let's have separate sections for methods (normal and remote), resources and fields to avoid going into 5 levels of subsections.